src/Features/RemotePaymentMethods/DefaultPaymentGateways.php

```diff
@@ -58,6 +58,17 @@ public static function get_all() {
 					self::get_rules_for_cbd( false ),
 				),
 			),
+			array(
+				'key'        => 'kco',
+				'title'      => __( 'Klarna', 'woocommerce-admin' ),
+				'content'    => __( 'Choose the payment that you want, pay now, pay later or slice it. No credit card numbers, no passwords, no worries.', 'woocommerce-admin' ),
+				'image'      => WC()->plugin_url() . '/assets/images/klarna-black.png',
+				'plugins'    => array( 'klarna-checkout-for-woocommerce' ),
+				'is_visible' => array(
+					self::get_rules_for_countries( array( 'SE', 'FI', 'NO' ) ),
```

Comment:
Is this supposed to be looking for the US or EU instead?

Reply:
@jeffstieler its not, these fallback gateways are the ones defined in the original payment methods list in JS (`getPaymentMethods`) so the "is visible" rules are copied from there.

When using the "is eu" check that they wrote I assumed it was gonna check for a european country and in other cases place the settings in US. I thought I had seen this working in development but I'll have to take another look. I'm definitely making some assumptions about the code in their plugin. I might have to see if there is a better approach.

Reply:
Good question.  I see more countries available for Klarna, but guess we're limiting to these 3 to show specific gateways.  These are the current used for Klarna in the task list, so I'm inclined to approve, but cc @pmcpinto for any background on why we only use these 3 countries.

Reply:
>  I see more countries available for Klarna, but guess we're limiting to these 3 to show specific gateways. These are the current used for Klarna in the task list, so I'm inclined to approve, but cc @pmcpinto for any background on why we only use these 3 countries.

We promote two different Klarna extensions: Checkout and Payments. Checkout should be displayed in 3 countries and Payments in 11. More info here: #5838